src/darwin/Framework/CHIP/MTRCluster.h

```diff
@@ -46,6 +47,29 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly) NSNumber * endpointID NS_REFINED_FOR_SWIFT MTR_NEWLY_AVAILABLE;
 @end
 
+/**
+ * Base for all MTRBaseCluster* types.
+ */
+MTR_NEWLY_AVAILABLE
+@interface MTRGenericBaseCluster : MTRCluster
+- (instancetype)init NS_UNAVAILABLE;
++ (instancetype)new NS_UNAVAILABLE;
+@end
+
+/**
+ * Base for all MTRCluster* types.
+ */
+MTR_NEWLY_AVAILABLE
+@interface MTRGenericCluster : MTRCluster
+- (instancetype)init NS_UNAVAILABLE;
```

Comment:
If init and new are already unavailable on MTRCluster you don’t need to repeat it here and below

Reply:
Done.